eve/io/mongo/mongo.py

```diff
@@ -80,6 +80,8 @@ class Mongo(DataLayer):
         'integer': lambda value: int(value) if value is not None else None,
         'float': lambda value: float(value) if value is not None else None,
         'number': lambda val: json.loads(val) if val is not None else None,
+        'boolean': lambda v:
+        {'1': True, 'true': True, '0': False, 'false': False}[str(v).lower()],
```

Comment:
1. We could also include things like "yes", "no", "on", "off", etc... But maybe it is overkill.
2. I would do something like {...}.get(str(v).lower()). That way we explicitly return None, if the value is not in that list. Currently we achieve the same thing indirectly by creating a KeyError, which then generates a None somewhere else. If it is explicit, there is less confusion when reading the code.